homeworks/roman.vlasov_batteryfree/homework_markup_1/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html leng="eu">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <title>HomeWork_MarkUp_1 - (Dreams)</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+<!--title page-->
+        <div class="title-page">
+             <a name="home"></a>
+            <header>
+                <img class="logo" src="images/logodreams.png" alt="logo">
+                <nav class="navigation">
+                    <ul class="link">
```

Comment:
here you giving a `link` class to the list, it's just a naming, but actually it could be something more descriptive, like `nav-list` or smth